### ISSUE TITLE
PTE::getElement(Position) broken

### ISSUE THREAD

**Issue by anhi** [BALL-Project/ball#75]
**Reported by Daniel Russel on 11 May 38777162 10:40 UTC**
It returns the ith Element alphabetically rather than in the periodic table.

'''Solution:''' Change the table that you look it up in.

**Comment by anhi** [comment]
**Commented by dstoeckel on 6 Aug 40864071 16:13 UTC**
More likely 2.0 as it breaks the existing interface.

**Comment by anhi** [comment]
**Commented by akdehof on 13 Sep 41131785 12:26 UTC**
Set to deprecated and add two new methods getElementByLexicographicIndex() (or similar) and getElementByAtomicNumber()

**Comment by anhi** [comment]
**Commented by Marcel on 27 Jul 41315910 17:37 UTC**
Well, I have to agree that this really was a _bug_ (and not simply a naming issue), for the following reasons:
-- PTE::getElement(Position) is used nowhere within BALL/BALLView (yep, I did check ;-) ), so that this bug had no ball-internal consequences
-- the previous documentation clearly indicated that the function was intended to look the element up by its atomic number; however, it did not do so!
-- who the heck would ever want to sort elements lexicographically and look them up this way? :-D 

Therefore, I now finally fixed this bug.
